code/mob/living/critter/bear.dm

```diff
@@ -1,23 +1,56 @@
 /mob/living/critter/bear
 	name = "space bear"
 	real_name = "space bear"
-	desc = "Oh god."
-	density = 1
+	desc = "WOORGHHH"
 	icon_state = "abear"
 	icon_state_dead = "abear-dead"
 	custom_gib_handler = /proc/gibs
 	hand_count = 2
-	can_throw = 1
-	can_grab = 1
-	can_disarm = 1
+	can_throw = FALSE
+	can_grab = TRUE
+	can_disarm = TRUE
 	blood_id = "methamphetamine"
 	burning_suffix = "humanoid"
+	skinresult = /obj/item/material_piece/cloth/leather
+	max_skins = 2
+	health_brute = 75
+	health_brute_vuln = 0.85
+	health_burn = 75
+	health_burn_vuln = 1.25
```

Comment:
Why the health buff?

Reply:
that was just the health values already in the file. Health might be a bit high but bears are pretty much only encountered in 1v1 and their arms are pretty weak in comparison to say brullbar arms.